lib/tpm2_eventlog_ascii.c

```diff
@@ -0,0 +1,208 @@
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <tss2/tss2_tpm2_types.h>
+
+#include "log.h"
+#include "tcg-efi-event.h"
+#include "tpm2_eventlog.h"
+#include "tpm2_eventlog_ascii.h"
+
+char* get_alg_name(UINT16 alg_id) {
```

Comment:
Any reason we cant use tpm2_alg_util_algtostr? Do you need the output to be "TPM2_ALG_SHA1" or is "sha1" good enough? If you need to full name can we smarten up that logic and add a flag to the routine to get the canonical name?

Reply:
No reason. Didn't see the `tpm2_alg_util` module when porting. Resolved.